packages/vue-app/template/store.js

```diff
@@ -3,6 +3,7 @@ import Vuex from 'vuex'
 
 Vue.use(Vuex)
 
+const log = (process.server ? require('consola') : console)
```

Comment:
`console` from server side is automatically redirected to consola. So no need to this.